From df02d4aadc7b155499d84165eb38a5e99fd71421 Mon Sep 17 00:00:00 2001
From: "hjnim.kim" <hjnim.kim@samsung.com>
Date: Thu, 11 Jul 2013 14:40:12 +0900
Subject: [PATCH] JQM closed popup does not add "visible" class when
 orientaiton change event is fired

JQM closed popup( this._isOpen = false ) does not add "ui-select-menu (visible class)" class when oriention change event is fired

Signed-off-by : Hyunjung Kim ( hjnim.kim@samsung.com )

Change-Id: I390be6236f2a1a41ef0c76b7889fe3e77b727c03
---
 libs/js/jquery-mobile-1.2.0/js/widgets/popup.js |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/js/jquery-mobile-1.2.0/js/widgets/popup.js b/libs/js/jquery-mobile-1.2.0/js/widgets/popup.js
index b593472..0bec1d4 100644
--- a/libs/js/jquery-mobile-1.2.0/js/widgets/popup.js
+++ b/libs/js/jquery-mobile-1.2.0/js/widgets/popup.js
@@ -109,7 +109,7 @@ define( [ "jquery",
 		},
 
 		_resizeTimeout: function() {
-			if ( !this._maybeRefreshTimeout() && this.positionTo === "window" ) {
+			if ( !this._maybeRefreshTimeout() && this.positionTo === "window" && this._isOpen ) {
 				// effectively rapid-open the popup while leaving the screen intact
 				this._trigger( "beforeposition" );
 				this._ui.container
-- 
1.7.9.5

